addons/mail/models/mail_message.py

```diff
@@ -742,6 +742,17 @@ def set_message_done(self):
             }
         )
 
+    def action_open_document(self):
```

Comment:
Another linting-guideline remark: could you move its definition before the "DISCUSS API" section comment ? It is not part of discuss specific methods, let us keep it after other methods.